docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,284 @@
+
+As a result, CA admins **register** users, a process in which the CA admin gives an enroll ID and secret (these are similar to a username and password) to an identity and assigns it a role and any required attributes. The CA admin then gives this enroll ID and secret to the ultimate user of the identity. The user can then execute a Fabric CA client command using this enroll ID and secret, returning the public/private key pair containing the role and attributes assigned by the CA admin.
+
+This process preserves both the integrity of the CA (because only CA admins can register users and assign roles and affiliations) and private keys (since only the user of an identity will have access to them).
+
+**While admin identities only need to be registered and enrolled with an "organization CA" that generates identity certificates for both admins and nodes alike, nodes must also be registered and enrolled with a TLS CA. This will create a public/private TLS key pair that nodes use to sign and encrypt their communications.** If the TLS CA has been created only using a "TLS" profile, the commands to register and enroll an identity with the organization CA are identical to those to register and enroll with the TLS CA. If you are using a CA that contains both profiles, you will have to specify the TLS profile when communicating with the CA. For more information about creating a CA that can only function as a TLS CA, check out the [CA deployment guide](cadeploy.html#modify-the-tls-ca-server-configuration).
+
+## Before you begin
+
+In this tutorial, we will assume that a CA server has been configured and set up [using the CA setup instructions](cadeploy,html). We will also show both the commands and variables used when registering an identity (a task handled by a CA admin or an identity with CA `registrar` rights), and when "enrolling" an identity (a task handled by the user of the identity).
+
+In either case, the `fabric-ca-client` must be set up, as it is used to make calls to the CA server where the identity is registered and enrolled. If you are operating in a production environment, you should have TLS enabled, and will need to provide TLS certificates to secure your communications with the CA. The TLS certificates will need to come from the TLS CA you spin up alongside the "organization" CA you use to generate identities, and this TLS CA is the same CA you will use to register and enroll node identities (since nodes use TLS to communicate with each other).
+
+Note: if you are connecting to different CA servers from the same CA client, when you use the `--mspdir` flag, you also **must** include the `-u` flag to target the correct CA server. If you will only being using a single CA client to target a single CA server, you have the option of specifying the CA server in the YAML file of the CA client.
```

Comment:
If you will only **`being`** using a single CA => If you will only **`be`** using a single CA